docs/reference/migration/migrate_7_0/settings.asciidoc

```diff
@@ -59,6 +63,7 @@ provided little value. The setting `http.pipelining.max_events` can still be use
 limit the number of pipelined requests in-flight.
 
 [float]
+//tag::notable-breaking-changes[]
```

Comment:
This one won't fail any upgrade if my understanding is correct, so I'd be leaning towards removing it.